quality/data_availability_dlt.sql

```diff
@@ -186,8 +192,46 @@ OR REFRESH LIVE TABLE data_availability USING DELTA AS (
   )
   SELECT
     t.country_name,
+    CASE 
+        WHEN t.country_name IN (
+            'Afghanistan',
+            'Albania',
+            'Armenia',
+            'Benin',
+            'Brazil',
+            'Burkina Faso',
+            'Burundi',
+            'Chile',
+            'Croatia',
+            'Colombia',
+            'Dominican Republic',
+            'Guatemala',
+            'Haiti',
+            'Kenya',
+            'Kiribati',
+            'Liberia',
+            'Mali',
+            'Mauritania',
+            'Mexico',
+            'Moldova',
+            'Niger',
+            'Paraguay',
+            'Peru',
+            'Poland',
+            'Senegal',
+            'Seychelles',
+            'Solomon Islands',
+            'South Africa',
+            'Togo',
+            'Tunisia',
+            'Uganda',
+            'Ukraine',
+            'Uruguay'
+        ) THEN 'Yes' ELSE 'No'
+    END AS boost_public,
     t.boost_earliest_year,
     t.boost_latest_year,
+    CASE WHEN m.country_name IS NULL THEN 'No' ELSE 'Yes' END as avail_on_mega,
```

Comment:
why not put avail_on_mega and boost_public side by side?

Reply:
Done!